src/server/report/app.js

```diff
@@ -369,12 +303,55 @@ class App extends LitElement {
 		searchParams.set('test', test.name);
 		return html`
 			<tr>
-				<th style="text-align: left;"><a href="${this._root}?${searchParams.toString()}" @click="${this._handleTestClick}" data-file="${file.name}" data-test="${test.name}">${test.name}</a></th>
+				<th style="text-align: left;"><a href="${this._root}?${searchParams.toString()}" @click="${this._handleListTestClick}" data-file="${file.name}" data-test="${test.name}">${test.name}</a></th>
 				${results}
 			</tr>
 		`;
 	}
-	_renderTestResults(fileData, testData) {
+	_renderMainView() {
+
+		if (this._filterFile === undefined) {
+			let list;
+			if (this._files.length === 0) {
+				list = html`<p>No tests exist for the selected filters.</p>`;
```

Comment:
Does this work here? I want more sad face magnifying glass.
```suggestion
				return this._renderEmpty();
```
If so, then probably some changes to the rest of this block.

Reply:
Thanks good catch -- yep more sad magnifying glass was required.